### PR TITLE
chore(docs): Remove user guide as part of move to new repo

### PR DESCRIPTION
**Pull request by Doug-AWS**
----

### Pull Request Checklist

* [ ] Testing
  - Unit test added
  - __CLI change?:__ manually run [integration tests](https://github.com/awslabs/aws-cdk/blob/master/packages/aws-cdk/integ-tests/test.sh) and paste output as a PR comment
  - __cdk-init template change?:__ coordinated update of integration tests with team
* [x] Docs
  - __jsdocs__: All public APIs documented
  - __README__: README and/or documentation topic updated
* [ ] Title and Description
  - __Change type__: title prefixed with **fix**, **feat** will appear in changelog
  - __Title__: use lower-case and doesn't end with a period
  - __Breaking?__: last paragraph: "BREAKING CHANGE: <describe what changed + link for details>"
  - __Issues__: Indicate issues fixed via: "**Fixes #xxx**" or "**Closes #xxx**"
* [ ] Sensitive Modules (requires 2 PR approvers)
  - IAM Policy Document (in @aws-cdk/aws-iam)
  - EC2 Security Groups and ACLs (in @aws-cdk/aws-ec2)
  - Grant APIs (only if not based on official documentation with a reference)

----

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license.

### CODE REVIEW

**Comment by Doug-AWS** [comment]
I wasn't sure what to do with the Reference section at the end of the tools, so I just left it and renamed the topic tool-ref.rst.

File: docs/src/index.rst [hunk 1]
```diff
@@ -12,106 +12,20 @@
...
-application synchronization (messaging and queuing).
-
-AWS uses a pay-as-you-go service model. You are charged only for the services that you |mdash| or
-your applications |mdash| use. Also, to make AWS useful as a platform for prototyping and
-experimentation, AWS offers a free usage tier, in which services are free below a certain level of
-usage. For more information about AWS costs and the free usage tier, see
-`Test-Driving AWS in the Free Usage Tier <http://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/billing-free-tier.html>`_.
-
-To obtain an AWS account, go to `aws.amazon.com <https://aws.amazon.com>`_ and click :guilabel:`Create an AWS Account`.
+This documentation is the refernce for the |cdk|.
```
Comment (eladb) [inline docs/src/index.rst]: typo

File: docs/src/tool-ref.rst [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+   License. A copy of the License is located at http://creativecommons.org/licenses/by-nc-sa/4.0/.
+
+   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+   either express or implied. See the License for the specific language governing permissions and
+   limitations under the License.
+
+.. _tool_ref:
+
+####################
+|cdk| Tool Reference
```
Comment (eladb) [inline docs/src/tool-ref.rst]: This seems like a duplicate (also exists on docs.aws)

File: docs/src/index.rst [hunk 3]
```diff
@@ -12,106 +12,20 @@
...
 
-   Getting Started <getting-started>
-   Tutorial <tutorial>
-   Examples <examples>
-   Concepts <concepts>
-   AWS Construct Library <aws-construct-lib>
-   AWS CloudFormation Library <cloudformation>
-   Tools <tools>
-   Writing Constructs <writing-constructs>
+   Tool Reference <tool-ref>
```
Comment (eladb) [inline docs/src/index.rst]: Let's just remove this from here and leave only the reference docs

**Comment by eladb** [comment]
Let's just delete the content that was migrated. I don't see value in maintaining a copy. We can always find it in the git history.

**Comment by RomainMuller** [comment]
I agree with @eladb. I feel `git`'s whole point is to keep an achieve of the previous state of everything, and it's pretty good at that. I would just drop the files and move on.

**Comment by eladb** [comment]
@Doug-AWS let's get this thing merged! I think we are ready

File: docs/src/index.rst (hunk 3 above)
Comment (Doug-AWS) [reply docs/src/index.rst]: Done

File: docs/src/tool-ref.rst (hunk 2 above)
Comment (Doug-AWS) [reply docs/src/tool-ref.rst]: I'll delete the file.

**Comment by Doug-AWS** [comment]
Okay, I blew away docs/archive

File: docs/src/index.rst [hunk 4]
```diff
@@ -12,106 +12,13 @@
...
+Welcome to the |cdk-long| (|cdk|)!
 
 The |cdk| is an infrastructure modeling framework that allows you to define your
 cloud resources using an imperative programming interface. *The CDK is currently
 in developer preview. We look forward to community feedback and collaboration*.
 
-To get started see :doc:`getting-started`
-
-.. image:: screencast.gif
-
```
Comment (eladb) [inline docs/src/index.rst]: Is this commented out?

File: docs/src/index.rst [hunk 5]
```diff
@@ -12,106 +12,12 @@
...
-   Getting Started <getting-started>
-   Tutorial <tutorial>
-   Examples <examples>
-   Concepts <concepts>
-   AWS Construct Library <aws-construct-lib>
-   AWS CloudFormation Library <cloudformation>
-   Tools <tools>
-   Writing Constructs <writing-constructs>
-   Reference <reference>
```
Comment (eladb) [inline docs/src/index.rst]: Didn't we intend to keep reference?

Comment (Doug-AWS) [reply docs/src/index.rst]: Yep, adding it back in.

File: docs/src/index.rst (hunk 4 above)
Comment (Doug-AWS) [reply docs/src/index.rst]: Yes, it's in the guide now: https://docs.aws.amazon.com/CDK/latest/userguide/what-is.html

**Comment by Doug-AWS** [comment]
I've merged master, so we should be good to go.

**Comment by Doug-AWS** [comment]
Well that was dumb of me. I'll add it back in.

**Comment by RomainMuller** [comment]
Added a merge commit from `master`, in hopes this makes the build pass (it was blocked due to an un-related integration test error).

**Pull request merged**